packages/cypress/tests/cypress/e2e/manual-snapshots.cy.ts

```diff
@@ -0,0 +1,6 @@
+it('Manual Snapshots / multiple snapshots are taken', () => {
+  cy.visit('/manual-snapshots');
+  cy.takeSnapshot('accordion collapsed');
```

Comment:
I'm wondering if we should have `cy.takeSnapshot()` take in an object so any possible future changes wouldn't be so disruptive. (e.g. it would be `cy.takeSnapshot({ name: 'a-snapshot-name' })`). We don't do this in Playwright, though (`name` is just one of the parameters passed in, not part of an object).

@tevanoff I'm leaning towards making this an object, but WDYT?

Reply:
That sounds like a good call. Could we have it accept either a string or an object with a name attribute to keep the API simple for most cases?

Reply:
Sounds good! I'll have it take in the object to be future-proof. Updating now.

Reply:
@tevanoff this is updated now, it's ready for your re-review.

Reply:
@skitterm Can it still accept a string instead of the full object to keep the simple API for most cases? I don't know if I like the more complicated API just in case we need to add more options later. Is that an option? And are you planning on doing the same in Playwright? I think maybe they should stay as consistent as possible.

Reply:
@tevanoff I see your point about the complexity, I hadn't thought that we could later accept `string | object`. I'll revert it back to accepting a string only, since that's all we need for now.

> And are you planning on doing the same in Playwright? I think maybe they should stay as consistent as possible.

I hadn't planned on changing Playwright's right now (since there are so many more Playwright users), but if we do the string way instead of object they should be similar enough (in Playwright we require the user to also pass along `testInfo` and `page`, which we don't need to pass along in Cypress, so they'll be a little different in that way).